internal/show/versions.go

```diff
@@ -67,19 +94,119 @@ func printDeploymentVersions(clusterInfo *cluster.Info, printer *table.Printer,
 		}
 
 		version, repository := images.ParseOperatorImage(deployment.Spec.Template.Spec.Containers[0].Image)
-		printer.Add(component, repository, version)
+
+		runningVersion, err := getVersionForComponent(clusterInfo, component,
+			labels.SelectorFromSet(deployment.Spec.Selector.MatchLabels).String())
+		if err != nil {
+			return err
+		}
+
+		printer.Add(component, repository, version, runningVersion)
 	}
 
 	return nil
 }
 
+func getVersionForComponent(clusterInfo *cluster.Info, component, labelSelector string) (string, error) {
```

Comment:
Wouldn’t it be better to use the `Selector` type for the label selector, rather than its string form? This would allow selector composition should we ever need that.

Reply:
👍 I suggested that too (our reviews crossed).

Reply:
I used string coz `ListOptions.LabelSelector` is string.

Reply:
Right, but that doesn’t mean we can’t keep the higher-level object here, and call `.String()` to provide the value for `LabelSelector`.